randovania/games/factorio/layout/factorio_preset_describer.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from randovania.layout.preset_describer import GamePresetDescriber
+
+if TYPE_CHECKING:
+    from randovania.layout.base.base_configuration import BaseConfiguration
+
+
+class FactorioPresetDescriber(GamePresetDescriber):
+    def format_params(self, configuration: BaseConfiguration) -> dict[str, list[str]]:
+        template_strings = super().format_params(configuration)
+        template_strings.pop("Logic Settings")
```

Comment:
remind me again what logic settings is used for? stuff like major/minor, avoid ponrs, have first progression locally etc.?

Reply:
trick settings